pyproject.toml

```diff
@@ -57,25 +56,25 @@ dynamic = ["version"]
 # pinned requirements files. See also `ci_requirements/README.txt`.
 
 dependencies = [
-  "astropy >= 5.1",
-  "h5py >= 3.7.0",
+  "astropy >= 5.2",
+  "h5py >= 3.8.0",
   # Minimum version of ipykernel should not exceed what's used by Google Colab
   "ipykernel >= 5.5.6",
   "ipywidgets >= 6.0.0",
   "lmfit >= 1.1.0",
-  "matplotlib >= 3.6.0",
-  "mpmath >= 1.2.1",
+  "matplotlib >= 3.7.0",
+  "mpmath >= 1.3.0",
   "numpy >= 1.24.0",
   "packaging >= 22.0",
-  "pandas >= 1.5.0",
+  "pandas >= 2.0.0",
   "requests >= 2.28.0",
-  "scipy >= 1.9.0",
-  "setuptools >= 61.2.0",
+  "scipy >= 1.10.0",
+  "setuptools >= 66.0.0",
   "tqdm >= 4.65.0",
   # Use loose requirements for voila because it impacts ipykernel version
   "voila >= 0.2.7",
-  "wrapt >= 1.14.0",
-  "xarray >= 2022.6.0",
+  "wrapt >= 1.15.0",
+  "xarray >= 2022.12.0",
 ]
```

Comment:
The updates to the minimum versions of dependencies are to drop minor releases that are $≥2$ years old, as per SPEC 0.